src/Okta.Sdk.IntegrationTests/PoliciesScenarios.cs

```diff
@@ -396,50 +396,27 @@ public async Task CreatePasswordPolicyRule()
                 Description = "The default policy applies in all situations if no other policy applies.",
             };
 
-            // TODO: Create an AddPasswordPolicyRuleOptions helper class with plain properties, encapsulate all the creation logic below and use this instead: client.Policies.AddPolicyRuleAsync(addPasswordPolicyRuleOptions);
             var createdPolicy = await client.Policies.CreatePolicyAsync(policy);
 
-            var policyRule = new PasswordPolicyRule()
+            var policyRuleOptions = new AddPasswordPolicyRuleOptions(createdPolicy.Id)
             {
                 Name = $"dotnet-sdk: CreatePasswordPolicyRule {guid}".Substring(0, 50),
-                Type = PolicyType.Password,
-                Conditions = new PasswordPolicyRuleConditions()
-                {
-                    People = new PolicyPeopleCondition()
-                    {
-                        Users = new UserCondition()
-                        {
-                            Exclude = new List<string>(),
-                        },
-                    },
-                    Network = new PolicyNetworkCondition()
-                    {
-                        Connection = "ANYWHERE",
-                    },
-                },
-                Actions = new PasswordPolicyRuleActions()
-                {
-                    PasswordChange = new PasswordPolicyRuleAction()
-                    {
-                        Access = "ALLOW",
-                    },
-                    SelfServicePasswordReset = new PasswordPolicyRuleAction()
-                    {
-                        Access = "ALLOW",
-                    },
-                    SelfServiceUnlock = new PasswordPolicyRuleAction()
-                    {
-                        Access = "DENY",
-                    },
-                },
+                UsersConditionExclude = new List<string>(),
```

Comment:
I wonder if we should add a prefix to the properties to indicate context, for example, instead of `UsersConditionInclude` rename it to `PeopleConditionUsersConditionInclude`? I believe it's clearer, you know exactly that the property is for PeopleConditions. Thoughts @andriizhegurov-okta @bryanapellanes-okta ?

Reply:
In this case it's more like `PeopleConditionUsersConditionsInclude` and so on, or in a more concise way `PeopleConditionsIncludeUsers` and other in the same class: `PeopleConditionsExcludeUsers`, `PeopleConditionsIncludeGroups`, `PeopleConditionsExcludeGroups`
I'm ok with this naming but since this is the only Users-to-Include list I'm not sure we these long prefixes are needed.

Reply:
It's more a matter of consistency. How do we usually name the properties in helpers?

Reply:
The idea is that mapping properties with our docs should be easy. Let's say I want to find this property in the [API reference](https://developer.okta.com/docs/reference/api/policy/#people-condition-object), as you can see it's under the `People object`, so I think a `People` prefix is helpful.

I like `PeopleConditionIncludeUsers` , I think it would be even more helpful if we add a `<remarks>` section in the comments with a link to the docs. For example:

```
/// <summary>
/// Gets or sets a list of users to be excluded
/// </summary>
<remarks>See <a href="https://developer.okta.com/docs/reference/api/policy/#people-condition-object"> the API documentation on policy objects. </remarks>
public IList<string> PeopleConditionIncludeUsers { get; set; }
```

Reply:
Done